demo/app-root.ts

```diff
@@ -1,19 +1,87 @@
-import { html, css, LitElement, customElement } from 'lit-element';
-import '../src/your-webcomponent';
+import { html, css, LitElement, customElement, query } from 'lit-element';
+import '../src/iaux-underlined-tab-bar';
+import type { UnderlinedTabBar } from '../src/iaux-underlined-tab-bar';
 
 @customElement('app-root')
 export class AppRoot extends LitElement {
+  @query('iaux-underlined-tab-bar') tabBar!: UnderlinedTabBar;
+
+  private itemClicked(e: CustomEvent) {
+    console.log(e);
+    console.log(e.detail.index);
+  }
+
+  private doSomething() {
+    this.tabBar.entries = [
+      { displayName: 'CHANGE' },
+      { displayName: 'TITLE' },
+      { displayName: 'OPTIONS' },
+      { displayName: 'ACITIVE' },
+    ];
+  }
+
+  private loadControl() {
+    console.log(this.tabBar.isLoading);
+    this.tabBar.isLoading = !this.tabBar.isLoading;
+  }
+
+  private indexSelect() {
```

Comment:
another naming question. how about `setRandomIndex()` or something similar that has a verb to indicate the action that is being performed?